x-pack/plugins/cases/server/client/cases/utils.ts

```diff
@@ -160,75 +145,80 @@ const addAlertMessage = (
 };
 
 export const createIncident = async ({
-  actionsClient,
   theCase,
   userActions,
   connector,
-  mappings,
   alerts,
   casesConnectors,
   userProfiles,
-}: CreateIncidentArgs): Promise<MapIncident> => {
-  const {
-    comments: caseComments,
-    title,
-    description,
-    created_at: createdAt,
-    created_by: createdBy,
-    updated_at: updatedAt,
-    updated_by: updatedBy,
-  } = theCase;
-
-  const params = { title, description, createdAt, createdBy, updatedAt, updatedBy };
+  publicBaseUrl,
+}: CreateIncidentArgs): Promise<ExternalServiceIncident> => {
   const latestPushInfo = getLatestPushInfo(connector.id, userActions);
   const externalId = latestPushInfo?.pushedInfo?.external_id ?? null;
-  const defaultPipes = externalId ? ['informationUpdated'] : ['informationCreated'];
-  let currentIncident: ExternalServiceParams | undefined;
 
   const externalServiceFields =
     casesConnectors.get(connector.actionTypeId)?.format(theCase, alerts) ?? {};
 
-  let incident: Partial<PushToServiceApiParams['incident']> = { ...externalServiceFields };
+  const connectorMappings = casesConnectors.get(connector.actionTypeId)?.getMapping() ?? [];
+  const descriptionWithKibanaInformation = addKibanaInformationToDescription(
+    theCase,
+    userProfiles,
+    publicBaseUrl
+  );
 
-  if (externalId) {
```

Comment:
There is no need to do a call to an external service. This was needed in case we need to append the case description to the description of the external service. We never supported this feature so I eliminated the code.